metrics.yaml

```
@@ -13,6 +13,24 @@
 ---
 $schema: moz://mozilla.org/schemas/glean/metrics/2-0-0
 
+rally:
+  id:
+    type: uuid
+    lifetime: user
+    send_in_pings:
+      - deletion-request
+      - rs01-event
+    description: |
+      The id of the Rally client. This is generated by
+      the Rally Core Add-on.
+    bugs:
+      - https://github.com/mozilla-rally/rally-core-addon/issues/117
+    data_reviews:
+      - https://github.com/mozilla-rally/rally-core-addon/pull/505#issuecomment-815826426
+    notification_emails:
+      - than@mozilla.com
```

```suggestion
      - hulmer@mozilla.com
```

For consistency, it should probably be my email address. Unless there is a reason @knowtheory was tagged, in which case, feel free to disregard this comment.